### ISSUE TITLE
Code should be readable and properly commented.

### ISSUE THREAD

**Issue by shresth2102001** [RocketChat/Rocket.Chat#21104]
Description
A code that is well formatted with proper indentation in accordance with conventional guidelines and proper commenting , is more readable. Going through the code, I found it as an enhancement, for all of us out there.
Steps to reproduce:
Read the code and rewrite it with a proper commenting and indentation

Expected behavior:
A code that is well formatted with proper indentation in accordance with conventional guidelines and proper commenting

Actual behavior:
No proper indentation and comments are there.

Server Setup Information:
Version of Rocket.Chat Server: 3.12.1
Operating System: macOS
Deployment Method: docker
Number of Running Instances: 1
Client Setup Information
Desktop App 3.1.1
Operating System: macOS 11.2.3

**Comment by Electron-2002** [comment]
How to go about this issue?  The codebase is very huge.

**Comment by shresth2102001** [comment]
I agree that might be difficult but it is a enhancement for the codebase.it can be slow time taking process.

**Comment by g-thome** [comment]
we use a linter to enforce formatting in general, but old code written before those rules got implemented may indeed contain inconsistencies. if you find any examples i ask you to report them so we can fix them

**Comment by KevLehman** [comment]
We applied strict eslint/prettier/typescript rules to the code since, so this shouldnt' be a problem anymore :)
